addons/policies/bootstrap_entity_policies.json

```diff
@@ -2860,6 +2860,31 @@
                     "entity-update"
                 ]
             }
+        },
+        {
```

Comment:
Can we move this to seperate file - "bootstrap_admin_policies.json"

Reply:
Done